debian/base/Dockerfile.rpm

```diff
@@ -0,0 +1,94 @@
+#
```

Comment:
i agree. i was trying not to upend the entire repo in one fell swoop. if you are ok with it, then im happy to :)